custom_components/unifiprotect/select.py

```diff
@@ -101,31 +121,31 @@ async def async_setup_entry(
     # Get Current Views
     liveviews = await upv_object.get_live_views()
 
-    select_entities = []
-    for item, item_type in SELECT_TYPES.items():
+    entities = []
+    for description in SELECT_TYPES:
         for device_id in protect_data.data:
-            if protect_data.data[device_id].get("type") in item_type[_SELECT_TYPE]:
-                select_entities.append(
+            if protect_data.data[device_id].get("type") in description.ufp_device_type:
```

Comment:
You can save some indent here by reversing the conditional to `not in` and doing a `continue`, then outdent below